### ISSUE TITLE
Demo project doesn't build

### ISSUE THREAD

**Issue by keith** [braintree/braintree_ios#538]
When building `Demo` on 50d28cca423cd5f2e96b3d5937b83c44f100c305 it fails with:

```
/Users/ksmiley/dev/braintree_ios/BraintreePaymentFlow/ThreeDSecure/BTThreeDSecureV2Provider.m:10:9: fatal error: 'CardinalMobile/CardinalMobile.h' file not found
#import <CardinalMobile/CardinalMobile.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
````

**Comment by scannillo** [comment]
Hi @keith - I'm not able to replicate this. Did you try re-running a `pod deintegrate` and `pod install`?

**Comment by keith** [comment]
Yep, this was actually a fresh clone with Xcode 11.5

**Comment by scannillo** [comment]
I am now able to replicate with a fresh clone. I added it to our list of items. Thank you for creating an issue!

**Comment by scannillo** [comment]
Hello. This was fixed in the latest release 4.36.0
